aiidalab_qe/pseudos.py

```diff
@@ -142,18 +147,21 @@ class PseudoFamilySelector(ipw.VBox):
         <div style="line-height: 140%; padding-top: 0px; padding-bottom: 10px; opacity:0.5;"><b>
         Exchange-correlation  functional</b></div>"""
     )
-
     dft_functional_help = ipw.HTML(
         """<div style="line-height: 140%; padding-top: 10px; padding-bottom: 10px; opacity:0.5;">
         The exchange-correlation energy is calculated based on the charge
         density using this functional. We currently provide support for two
         well-established generalised gradient approximation (GGA) functionals:
         PBE and PBEsol.</div>"""
     )
-
     installed = traitlets.Bool()
+    disable = traitlets.Bool()
```

Comment:
This should be:
```suggestion
    disabled = traitlets.Bool()
```
And are you actually using that trait?

Reply:
No. But didn't you say we needed the `disabled` trait? 😭 

At some point I thought I knew why, so I reinstated it 😎

Reply:
> No. But didn't you say we needed the `disabled` trait? 😭
> 
> At some point I thought I knew why, so I reinstated it 😎

The `disabled` trait should toggle the widget into a "disabled" state when set to True. What that exactly means depends a little bit on the widget so that's why there is no default implementation for this. Ideally, subsequent steps cannot be modified (i.e. are disabled) until the previous step has been confirmed. Hence why most widgets within the app should have this trait and also implement the corresponding behavior.

Reply:
Thanks for the explanation :). Makes sense!